services/ui-src/src/components/overlays/EntityDetailsOverlay.tsx

```diff
@@ -169,7 +169,7 @@ const sx = {
         whiteSpace: "break-spaces",
         fontSize: "xl",
         lineHeight: "1.75rem",
-        "&:first-child": {
+        "&:first-of-type": {
```

Comment:
Fixes `The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".`

Reply:
🤔 we don't do server-side rendering but I guess it's just saying to watch out for it